action/container/container.go

```diff
@@ -117,13 +131,47 @@ func Setup(ctx context.Context, client *dagger.Client, opts *SetupOpts, dockerfi
 	//   WithMountedDirectory
 	//     Create a OverlayFS with bottom layer Read-Only
 	//     Directory in container must exist
-	container, err = container.
+	container = container.
 		WithExec([]string{"mkdir", "-p", opts.MountContainerDir}).
 		WithMountedDirectory(
 			opts.MountContainerDir,
 			client.Host().Directory(opts.MountHostDir)).
-		WithWorkdir(opts.WorkdirContainer).
-		Sync(ctx)
+		WithWorkdir(opts.WorkdirContainer)
+
+	// Get current working directory
+	pwd, err := os.Getwd()
+	if err != nil {
+		slog.Error(
+			"Could not get working directory, should not happen",
```

Comment:
Isn't the "should not happen" duplicate with the suggestion message?

Reply:
Now that you mention it ... yes it is.